src/navigator/Navigator.tsx

```diff
@@ -538,6 +539,7 @@ const generalScreens = (Navigator: typeof Stack) => (
 
 const earnScreens = (Navigator: typeof Stack) => (
   <>
+    <Navigator.Screen name={Screens.EarnHome} component={EarnHome} options={headerWithBackButton} />
```

Comment:
use custom header over nav header to avoid the header not greying out when the bottom sheet is open

Reply:
I'm having some trouble getting a custom header to work with the `useScrollAwareHeader` to have scrolling work and the title show up on the top. Is there any example of getting these to work together that we have, I'm having trouble finding one.

Reply:
`useScrollAwareHeader` is meant to work with nav header and not custom header. We'll likely need a new wrapper for custom headers, which is beyond the scope of this. Let's make an issue